libs/application-generic/src/usecases/workflow/update-workflow/update-workflow.usecase.ts

```diff
@@ -85,7 +85,7 @@ export class UpdateWorkflow {
       updatePayload.name = command.name;
     }
 
-    if (command.active) {
+    if (command.active !== undefined && command.active !== null) {
```

Comment:
`command.active` is never `null`.

```suggestion
    if (command.active !== undefined) {
```